app/services/webhooks/event_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Webhooks
+  class EventService < Webhooks::BaseService
+    private
+
+    def current_organization
+      @current_organization ||= Organization.find(object[:organization_id])
+    end
+
+    def object_serializer
+      ::ErrorSerializer.new(
+        OpenStruct.new(object),
+        root_name: 'error_event',
```

Comment:
`object_type` and `root_name` should have the same value.

The idea behind this is that the webhook consumer should be able to know easily what attribute to look for in the payload:
```json
{
  "object_type": "event_error",
  "event_error": {
   ...
  }
```

Reply:
Thanks for the clarification, wasn't aware of it :)